.circleci/config.yml

```diff
@@ -142,23 +145,23 @@ jobs:
             kubectl annotate -n c100-application-<< parameters.env_name >> \
                     deployment/c100-application-deployment-<< parameters.env_name >> \
                     kubernetes.io/change-cause="${BUILD_DATE} set image ${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1} via CircleCI"
- 
-  deploy_image_to_live:
+
+  deploy_image_to_live_prod:
     <<: *cloud_container
     parameters:
       env_name:
-        type: string
```

Comment:
I think is this. Leave it as `string` do not change to `production`.

Reply:
Resolved